components/treeview/drag-drop.md

```diff
@@ -28,8 +31,35 @@ To enable the Drag and Drop functionality:
 
 1. Set the `Draggable` parameter of the `<TelerikTreeView>` to `true`
 
-1. Use the `OnDrop` event to handle the drag and drop operations and modify the data source as per your business logic.
+1. Use the Drag events to handle the drag and drop operations and modify the data source as per your business logic.
 
+## DragStart Event
+
+The `DragStart` event fires when the user starts dragging a node. It provides details for the dragged items and allows you to cancel the event.
+
+### Event Arguments
+
+The `DragStart` event provides an object of type `TreeViewDragStartEventArgs` to it's event handler which exposes the following fields:
+
+* `Item` - an object you can cast to your model class to obtain the current data item.
```

Comment:
For consistency `OnDrop` section, use a table to represent the event arguments.

Reply:
Wouldn't three tables of this size bloat the page too much? Taking consistency into account, maybe I should change the OnDrop instead.

Reply:
Tables are fine, don't worry about bloating.